pkg/haproxy/instance_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/kylelemons/godebug/diff"
 	yaml "gopkg.in/yaml.v2"
 
+	ingtypes "github.com/jcmoraisjr/haproxy-ingress/pkg/converters/ingress/types"
```

Comment:
This adds a layer violation, haproxy package doesn't know about converters. Removing this will also fix the circular reference. See also the comment below about importing the const.